doc/examples/tracking_tissue_classifier.py

```diff
@@ -188,9 +189,9 @@
          affine,
          labels.shape)
 
-streamlines = [sl for sl in all_streamlines_binary_classifier]
+streamlines_generator = Streamlines(all_streamlines_binary_classifier)
 fvtk.clear(ren)
-fvtk.add(ren, fvtk.line(streamlines, line_colors(streamlines)))
+fvtk.add(ren, fvtk.line(streamlines_generator, line_colors(streamlines_generator)))
```

Comment:
Same comment here -- should probably be `streamlines`, instead of `streamlines_generator`?